packages/framework/src/Foundation/Internal/LoadYamlConfiguration.php

```diff
@@ -43,6 +44,10 @@ public function bootstrap(Application $app): void
     protected function mergeParsedConfiguration(): void
     {
         foreach ($this->yaml->getData() as $namespace => $data) {
+            if ($namespace === 'hyde' && isset($data['authors'])) {
+                $data['authors'] = $this->parseAuthors($data['authors']);
+            }
```

Comment:
Thinking if we should merge in authors defined in the `hyde.php` config, but I don't think we should, because what if the users wants to remove the default entries present there? Makes more sense to just keep it simple and override everything.

Reply:
Draft: 208c84834f7b3d159080e4ec7b402f79a344de1a

Reverted: 5f1e2d360165f82bba25395e9ce784e2e6e94a6d